doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -28,7 +28,11 @@ To enable a PSA driver, set the following configurations:
 
 If multiple drivers are enabled, the first ordered item in this table takes precedence for an enabled cryptographic feature, unless the driver does not enable or support it.
 
-Enabling or disabling PSA driver specific configurations controls the support for a given algorithm, per driver.
+The configuration granularity of the driver :ref:`nrf_security_drivers_cc3xx` allows enabling or disabling of specific PSA APIs (such as psa_cipher_encrypt, psa_sign_hash), but not individual algorithms.
```

Comment:
```suggestion
The driver :ref:`nrf_security_drivers_cc3xx` allows enabling or disabling of specific PSA APIs (such as psa_cipher_encrypt, psa_sign_hash), but not individual algorithms.
```

It read weird to me. With the new phrasing the two sentences are more similar. It is not clear if it was intentional to communicate that they are different somehow by using different phrasing.